go/mysql/flavor_mysql.go

```diff
@@ -32,7 +32,7 @@ type mysqlFlavor struct{}
 
 // masterGTIDSet is part of the Flavor interface.
 func (mysqlFlavor) masterGTIDSet(c *Conn) (GTIDSet, error) {
-	qr, err := c.ExecuteFetch("SELECT @@GLOBAL.gtid_executed", 1, false)
+	qr, err := c.ExecuteFetch("SELECT @@global.gtid_executed", 1, false)
```

Comment:
any particular reason you needed to change this?

Reply:
ripple does not like `GLOBAL` , I think we should add some failsafe way to fix the case sensitiveness.